js/ui/popup.js

```diff
@@ -404,6 +407,7 @@ var Popup = Overlay.inherit({
         var isFullscreen = this.option("fullScreen");
 
         this._toggleFullScreenClass(isFullscreen);
+        this._toggleContentAutoResizableClass();
```

Comment:
Render calls the `_renderGeometry` method. This call can be in the top of the `renderGeometry`